roles/scalelite/defaults/main.yml

```diff
@@ -0,0 +1,207 @@
+scalelite_nginx_dhparam_size: 4096
+scalelite_nginx_worker_processes: auto
+scalelite_nginx_ssl: true
+scalelite_nginx_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 15M
+  limits:
+    cpus: "{{ ansible_facts['processor_vcpus'] / 2 }}"
+    memory: "{{ ansible_facts['memory_mb']['real']['total'] / 2 }}M"
+scalelite_nginx_metrics_endpoints:
+  - name: node-exporter
+    public_port: 9100
+    private_port: 9099
+  - name: bbb-autoscaler
+    public_port: 9200
+    private_port: "{{ scalelite_autoscaler_config.metrics_exporter_port }}"
+  - name: cadvisor
+    public_port: 9300
+    private_port: 9299
+  - name: postgres-exporter
+    public_port: 9188
+    private_port: 9187
+scalelite_nginx_logs_folder: /var/log/nginx
+
+scalelite_user_id: 1000
+scalelite_group_id: 1000
+scalelite_web_concurrency: 4
+scalelite_rails_max_threads: 4
+scalelite_log_level: warn
+scalelite_multitenancy_enabled: true
+scalelite_version: v1.5.1.7-focal260-alpine
+# scalelite_secret_key_base:
+# scalelite_api_key
+scalelite_poller_image: "blindsidenetwks/scalelite:{{ scalelite_version }}-poller"
+scalelite_poller_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 100M
+  limits:
+    cpus: "{{ ansible_facts['processor_vcpus'] / 2 }}"
+    memory: "{{ ansible_facts['memory_mb']['real']['total'] / 2 }}M"
+scalelite_api_image: "blindsidenetwks/scalelite:{{ scalelite_version }}-api"
+scalelite_api_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 250M
+  limits:
+    cpus: "{{ ansible_facts['processor_vcpus'] / 2 }}"
+    memory: "{{ ansible_facts['memory_mb']['real']['total'] / 2 }}M"
+
+scalelite_hostname: "{{ inventory_hostname }}"
+scalelite_letsencrypt_folder: /etc/letsencrypt
+scalelite_terraform_workspace: prod
+scalelite_bbb_dialin_enabled: false
+scalelite_metrics_basic_auth_username: metrics-user
+# scalelite_metrics_basic_auth_password:
+scalelite_security_txt: |
+  # Security.txt content
+
+scalelite_autoscaler_user_id: 65534
+scalelite_autoscaler_group_id: 65534
+scalelite_autoscaler_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 20M
+  limits:
+    cpus: '0.5'
+    memory: 100M
+scalelite_autoscaler_datacenter_ids: []
+scalelite_autoscaler_server_name_regex: "bbb-.*"
+# scalelite_autoscaler_ionos_username:
+# scalelite_autoscaler_ionos_password:
+# scalelite_autoscaler_ionos_contract_id:
+# scalelite_autoscaler_bbb_api_key:
+# scalelite_autoscaler_grafana_token:
+scalelite_autoscaler_bbb_resources:
+  cpu:
+    min_cores: 8
+    max_cores: 16
+    max_usage: 0.7
+  memory:
+    min_bytes: 16384
+    max_bytes: 32768
+    max_usage: 0.7
+scalelite_autoscaler_waiting_time: 60
+# scalelite_autoscaler_grafana_endpoint: https://grafana.example.com/api/datasources/proxy/uid/UUID/
+scalelite_autoscaler_metrics_private_port: 9199
+scalelite_autoscaler_config:
+  app_name: bbb-scaler
+  stage: prod
+  scaling_mode: heuristic
+  service_type: BBB
+  provider_type: Ionos
+  metrics_source_type: Prometheus
+  ionos_config:
```

Comment:
Can you leave the provider specific variables out?

Reply:
Done